### ISSUE TITLE
ataru ignores code samples including an empty line

### ISSUE THREAD

**Issue by madziaf** [CodePadawans/ataru#34]
I create a file bug1.md file:
```
1

2
```
I run it with ataru and get this output:

```
Run options: --seed 1432

# Running:



Finished in 0.000990s, 0.0000 runs/s, 0.0000 assertions/s.

0 runs, 0 assertions, 0 failures, 0 errors, 0 skips
```

No test was run. I would expect one test to run.

**Comment by schultyy** [comment]
When you fix the bug, also write a test. It proves that you fixed the bug. Also then you see if this bug appears again in the future, because then the test fails.

**Comment by madziaf** [comment]
check the TestChanger and test for it on the branch 30 :)

**Comment by madziaf** [comment]
https://github.com/gettalong/kramdown/issues/165

**Comment by nanoparsec** [comment]
Finally solved <3 with pull request #41
